libcudacxx/include/cuda/__iterator/counting_iterator.h

```diff
@@ -301,6 +292,31 @@ struct counting_iterator : public __counting_iterator_category<_Start>
     return *this;
   }
 
+  //! @brief Creates a copy of a @c counting_iterator advanced by a given number of elements
+  //! @param __iter The @c counting_iterator to advance
+  //! @param __n The amount of elements to advance
+  _CCCL_TEMPLATE(class _Start2 = _Start)
+  _CCCL_REQUIRES(__advanceable<_Start2>)
+  [[nodiscard]] _CCCL_API friend constexpr counting_iterator
+  operator+(counting_iterator __iter, difference_type __n) noexcept(::cuda::std::__integer_like<_Start2>)
+  {
+    __iter += __n;
+    return __iter;
+  }
+
+  //! @brief Creates a copy of a @c counting_iterator advanced by a given number of elements
+  //! @param __iter The @c counting_iterator to advance
+  //! @param __n The amount of elements to advance
+  _CCCL_TEMPLATE(class _Start2 = _Start)
+  _CCCL_REQUIRES(__advanceable<_Start2>)
+  [[nodiscard]] _CCCL_API friend constexpr counting_iterator
+  operator+(difference_type __n, counting_iterator __iter) noexcept(::cuda::std::__integer_like<_Start2>)
+  {
+    return __iter + __n;
+  }
+
+  //! @brief Decrements a @c counting_iterator by a given number of elements
```

Comment:
Not sure but it should be consistently "increments/decrements **by** a given number of elements" or "increments/decrements a given number of elements" in all files.
```suggestion
  //! @brief Decrements a @c counting_iterator a given number of elements
```